score_and_rank_contributors.py

```diff
@@ -0,0 +1,74 @@
+#! /usr/bin/env python
```

Comment:
The whitespace after shabang (`#!`) is unnecessary, right? If so, it should be removed